Paraswap-Agents/AdminOperations/src/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { Interface } from "ethers/lib/utils";
+
+// array of operations to monitor
+export const ADMIN_OPERATIONS = [
+  "function transferTokens(address token, address payable destination, uint256 amount)",
+  "function setImplementation(bytes4 selector, address implementation)",
+  "function registerPartner(address partner, uint256 _partnerShare, bool _noPositiveSlippage, bool _positiveSlippageToUser, uint16 _feePercent, string calldata partnerId, bytes calldata _data)",
+  "function setFeeWallet(address payable _feeWallet)",
+  "event RouterInitialized(address indexed router)",
+  "event AdapterInitialized(address indexed adapter)",
+];
+
+export const AUGUSTUS_SWAPPER_CONTRACT: string = "0xDEF171Fe48CF0115B1d80b88dc8eAB59176FEe57";
+export const SWAPPER_IFACE = new Interface(ADMIN_OPERATIONS);
```

Comment:
Maybe this declaration could be in the `agent.spec.ts` file, since `SWAPPER_IFACE` is only used there and requires an `ethers` import?